README.md

```diff
@@ -8,3 +8,10 @@ Whole projects was written using technologies, concepts and patterns such as:
 - Walking Skeleton
 - Mockito
 - TestNG
+- Maven
+
+
+## How to run ?
+1. Clone this repository and go to main directory
+2. Build project by: mvn clean install
```

Comment:
mvn clean install  -> ``mvn clean install`` or ```mvn clean install``` will be much clearer. The same situation for mvn clean install. Example syntax in README.md  ('How to run' paragraph) https://github.com/kukulastudent/crawler .